lib/github_s.mli

```diff
@@ -1062,19 +1062,20 @@ module type Github = sig
       name:string ->
       unit ->
       unit Response.t Monad.t
-    (** [remove_label ~user ~repo ~num ~name ()] removes the label [name]
-        from the issue [num] in the repo [user]/[repo]. *)
+    (** [remove_label ~user ~repo ~num ~name ()] activates after the
```

Comment:
The function returns a `Github.Monad.t` value which is fulfilled after the removal occurs. This diff just updates the docs to be in line with the rest of the interface docs. I'm happy to change the wording everywhere if you have a better suggestion.